main.tf

```
@@ -22,6 +22,15 @@ resource "aws_lb" "this" {
     }
   }
 
+  dynamic "connection_logs" {
+    for_each = length(var.connection_logs) > 0 ? [var.connection_logs] : []
+    content {
+      bucket  = connection_logs.value.bucket
+      enabled = try(connection_logs.value.enabled, false)
```

If we default to `true` then users just need to supply details and its enabled - making the happy path shorter
```suggestion
      enabled = try(connection_logs.value.enabled, true)
```